backend/shared/getCurrentTimePeriod.js

```diff
@@ -0,0 +1,14 @@
+const moment = require('moment')
```

Comment:
On the off chance you don't know, there is a version of this in backend/utils.js